### PR TITLE
Various cosmetic improvements

### PR DESCRIPTION
**Pull request by alexreg**
Related to the larger effort of https://github.com/rust-lang/rust/pull/58036.

### CODE REVIEW

File: crates/stdsimd-test/src/lib.rs [hunk 1]
```diff
@@ -136,17 +136,16 @@ pub fn assert(fnptr: usize, fnname: &str, expected: &str) {
...
                 "cpuid" => 30,
 
-                // Apparently on Windows LLVM generates a bunch of
+                // Apparently, on Windows, LLVM generates a bunch of
                 // saves/restores of xmm registers around these
-                // intstructions which blows the 20 limit
-                // below. As it seems dictates by Windows's abi
-                // (I guess?) we probably can't do much
-                // about it...
+                // intstructions which exceeds the 20 limit
```
Comment (gnzlbg) [inline crates/stdsimd-test/src/lib.rs]: shouldn't this be "exceed" ? we are talking about "the instructions" and not "LLVM"

File: crates/core_arch/src/x86_64/xsave.rs [hunk 2]
```diff
@@ -105,7 +105,7 @@ pub unsafe fn _xsaves64(mem_addr: *mut u8, save_mask: u64) {
     xsaves64(mem_addr, (save_mask >> 32) as u32, save_mask as u32);
 }
 
-/// Perform a full or partial restore of the enabled processor states using the
+/// Performs a full or partial restore of the enabled processor states using the
```
Comment (gnzlbg) [inline crates/core_arch/src/x86_64/xsave.rs]: Below this PR changes "Get" to "Gets", why go the other way around here?

File: crates/stdsimd-test/src/lib.rs (hunk 1 above)
Comment (alexreg) [reply crates/stdsimd-test/src/lib.rs]: I think the implicit noun here is "bunch"?

File: crates/core_arch/src/x86_64/xsave.rs (hunk 2 above)
Comment (alexreg) [reply crates/core_arch/src/x86_64/xsave.rs]: Wait, is this the other way round...? I thought I changed imperative to indicative everywhere. Maybe you could point out the other case, sorry.

**Comment by alexreg** [comment]
@gnzlbg Thanks for looking at this. See my above comments, but I think we're basically good to go.

File: crates/core_arch/src/x86_64/xsave.rs (hunk 2 above)
Comment (gnzlbg) [reply crates/core_arch/src/x86_64/xsave.rs]: See https://github.com/rust-lang-nursery/stdsimd/pull/688/files#diff-e6a419cbbe65f76d6e34e2cff40c3d6fR93

Comment (alexreg) [reply crates/core_arch/src/x86_64/xsave.rs]: Oh. Well these are within bodies. The convention seems to be to use the indicative mood for function/method comments, but the imperative mood for (ordinary) comments within bodies.

**Comment by alexreg** [comment]
Any idea why AppVeyor is failing? (Travis is fine.)

**Comment by gnzlbg** [comment]
> Any idea why AppVeyor is failing?

Something changed in rust-lang/rust upstream, and some of our test suite broke. Don't have access to a Windows machine, haven't had much time to investigate why yet, and playing code golf on Appveyor CI isn't particularly fun.

**Comment by alexreg** [comment]
@gnzlbg Oh okay. Shall we merge this regardless then?

**Comment by gnzlbg** [comment]
It has already been merged, thanks! ;)

**Comment by alexreg** [comment]
@gnzlbg Hah sorry, I'm so used to seeing the usual Bors rigmarole, I didn't notice. Thanks. :-)

**Comment by gnzlbg** [comment]
Yeah, we don't use bors here (yet).

**Pull request merged**